### ISSUE TITLE
Update http metric spec

### ISSUE THREAD

**Issue by gfuller1** [open-telemetry/opentelemetry-specification#897]
Context: to prevent #739 from being blocked indefinitely we plan to merge that and have a followup PR to update it once the work for #855 is done. 

The work in the http metric spec that would need to be done once #855 is implemented is the following:
- convert the table of labels in the http metrics to references to the common label/attribute list
- clarify which common labels are suggested for http metrics
- add a note describing the cardinality effects of the labels we are using and the impact of adding extra labels not specified on the spec
- give examples of common labels that are on http span events that should not be on http metric events

**Comment by ocelotl** [comment]
I asked Claude to review this issue and Claude suggests it be closed with this explanation:

> Issue is purely about HTTP metric semantic conventions: converting the metric label table to references to the common attribute list, listing suggested labels, and adding cardinality notes. HTTP semantic conventions (including metrics with attribute references and cardinality guidance) moved to open-telemetry/semantic-conventions and are maintained there, not in this spec repo. Should be tracked in that repo if still needed.

To guard against a mistaken suggestion, Claude then independently re-verified this conclusion:

> 2020 p3 editorial issue (no comments, no closing PR) asking to convert HTTP metric label tables to common-attribute references + add cardinality notes; HTTP metric semantic conventions moved to open-telemetry/semantic-conventions where this work now lives (issue follows up on merged #739). Moved-repo verdict holds.

Claude evaluated this issue with medium confidence that it can be closed.

Claude has classified this issue as belonging in another repository.

The intention of this comment is to make it easier for a human being to review this issue before making a decision.

**Comment by svrnm** [comment]
@gfuller1 can you take a look at @ocelotl comment and make a call if this is the correct analysis and this can be closed? Thanks.
